.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: release
+
+on: workflow_dispatch 
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      # setup
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+
+      # build resources
+      - run: node ./dev/icon_generator/makeicons
+      - run: node ./dev/resource_packager
+
+      # publish
+      # TODO: cleaner version number
+      - name: get version
+        id: version
+        run: |
+          echo ::set-output name=current-version::$(cat ./editor/script/engine/bitsy.js | grep -oP '(?<=major: )\d+').$(cat ./editor/script/engine/bitsy.js | grep -oP '(?<=minor: )\d+')
+      - name: publish to itch.io
+        uses: josephbmanley/butler-publish-itchio-action@v1.0.3
+        env:
+          BUTLER_CREDENTIALS: ${{ secrets.BUTLER_CREDENTIALS }}
```

Comment:
this is a secret which will need to be configured under https://github.com/le-doux/bitsy/settings/secrets/actions > "New repository secret"; the name should be `BUTLER_CREDENTIALS`, and the value should be an API key generated from your itch account, which you can create/find under https://itch.io/user/settings/api-keys

[reference](https://github.com/manleydev/butler-publish-itchio-action#environment-variables)